packages/ui/src/validators/components/nominator/NominatorItems.tsx

```diff
@@ -0,0 +1,954 @@
+          )}
+        </ControllerCell>
+
+        <RewardsCell>
+          <TextSmall>{rewardDestination || '-'}</TextSmall>
+        </RewardsCell>
+
+        <StakeCell>
+          <StakeInfo>
+            <StakeRow>
+              <TokenValue value={position.activeStake} />
+            </StakeRow>
+            {unlockingTotal.gt(BN_ZERO) && (
+              <StakeRow>
+                {getUnbondingTimeInfo.hasUnbonding && !getUnbondingTimeInfo.isRecoverable && unbondingTooltipText && (
+                  <Tooltip popupContent={unbondingTooltipText}>
+                    <UnbondingClockIcon>
+                      <WatchIcon />
+                    </UnbondingClockIcon>
+                  </Tooltip>
+                )}
+                {getUnbondingTimeInfo.hasUnbonding && getUnbondingTimeInfo.isRecoverable && (
+                  <RecoverableButton
+                    size="small"
+                    square
+                    onClick={(e) => {
+                      e.stopPropagation()
+                      openManageActionModal('withdraw')
+                    }}
+                    title="Recoverable - click to withdraw"
+                  >
+                    <LockSymbol />
+                  </RecoverableButton>
+                )}
+                <TokenValue value={unlockingTotal} />
+                <TextSmall lighter>Unbonding:</TextSmall>
+              </StakeRow>
+            )}
+          </StakeInfo>
+        </StakeCell>
+
+        <AssignmentsCell>
+          {position.role === 'nominator' && position.nominations.length > 0 ? (
+            <Tooltip
+              popupContent={
+                <NominationsTooltipContent>
+                  {nominationsInfo === undefined ? (
+                    <TextSmall lighter>Loading...</TextSmall>
+                  ) : nominationsInfo && nominationsInfo.length > 0 ? (
+                    <>
+                      {nominationsInfo.some((n) => n.isActive) && (
+                        <>
+                          <TooltipSection>
+                            <TooltipPopupTitle>
+                              Active ({nominationsInfo.filter((n) => n.isActive).length})
+                            </TooltipPopupTitle>
+                            {nominationsInfo
+                              .filter((n) => n.isActive)
+                              .map((nom) => {
+                                try {
+                                  if (!nom || !nom.address) {
+                                    error('Invalid nom object:', nom)
+                                    return null
+                                  }
+                                  return (
+                                    <TooltipRow key={nom.address}>
+                                      <TooltipText>{shortenAddress(encodeAddress(nom.address), 20)}</TooltipText>
+                                      {nom.stake && (
+                                        <TooltipText>
```

Comment:
Please try my change check how to widen the tooltip to fit the whole row.
<img width="333" height="325" alt="toolt" src="https://github.com/user-attachments/assets/ace4ce64-2320-419c-a860-2a95053162e3" />

Reply:
All values show 0.